sips/msg_struct_encoding.md

```diff
@@ -37,6 +37,12 @@ SSZ is 66% smaller (in the test, 1,640 bytes vs 3,700) vs json, using snappy com
 
 SSZ hash tree root enables substituting full data objects with their hash tree root without changing the full message hash tree root which also doesn’t change the BLS signature signing the object. Below we explain how this property is used.
 
+**Compression**  
+We’ve chosen to use [S2](https://github.com/klauspost/compress/tree/master/s2#snappy-compatibility) compression for SSZ encoding, based on the benchmark described below.
```

Comment:
Are we talking about 10% diff between snappy and s2? For interoperability sake isn’t it worth staying with snappy?
How well tested and used s2 is? Who is behind it?

Reply:
I have a dedicated task for this.
Once the spec will be aligned with the new message structs, we can benchmark the real data and have a proper results & decisions.

Reply:
I mean the diff between s2 and snappy?